tests/sentry/incidents/test_action_handlers.py

```diff
@@ -122,6 +122,19 @@ def test(self):
         }
         assert expected == handler.generate_email_context(status)
 
+    def test_environment(self):
+        status = TriggerStatus.ACTIVE
+        environments = [
+            self.create_environment(project=self.project, name="prod"),
+            self.create_environment(project=self.project, name="dev"),
+        ]
+        alert_rule = self.create_alert_rule(environment=environments)
+        alert_rule_trigger = self.create_alert_rule_trigger(alert_rule=alert_rule)
+        action = self.create_alert_rule_trigger_action(alert_rule_trigger=alert_rule_trigger)
+        incident = self.create_incident()
+        handler = EmailActionHandler(action, incident, self.project)
+        assert "prod, dev" == handler.generate_email_context(status).get("environment")
```

Comment:
This test might end up unstable if the db order arbitrarily changes, might be a good idea to sort just to handle that case.